# PSTricksパッケージを使用する場合の修正
# 
#  PSTricksパッケージの pst-text.sty に含まれている \pstextpath マクロ中で
#  日本語を使用するとPostScriptのRIPがエラーを返します。
#  これはPSTricksのヘッダファイル pst-text.pro が1バイトのフォントにしか対応
#  していないためです。
#  日本語のような2バイトフォントに対処するため、このパッチファイルを
#  pst-text.pro に当ててください。
#  なお、このパッチは PSTricks Version 97 に対するものです。
#  PSTricksパッケージを使用しない場合はこの修正は必要ありません。
#
#  pst-text.pro が置いてあるディレクトリ（$(TEXMF)/dvips/pstricks/ 付近？）に
#  移動し、そこにこのファイルをコピーします。それから
#
#    patch < PSTricks.jpatch
#
#  を実行してください。
#
*** pst-text.pro.orig   Tue Jan 25 01:45:05 2000
--- pst-text.pro        Tue Jan 25 02:19:40 2000
***************
*** 50,58 ****
  /TextPathShow
  { /String exch def
    /CharCount 0 def
!   String length
!   { String CharCount 1 getinterval ShowChar
!     /CharCount CharCount 1 add def
    } repeat
  } def
  
--- 50,63 ----
  /TextPathShow
  { /String exch def
    /CharCount 0 def
!   /CharSize 1 def
!   currentfont /FontType get 0 eq
!   { currentfont /FMapType get dup 2 eq exch dup 5 eq exch 9 eq or or
!     { /CharSize 2 def} if
!   } if
!   String length CharSize idiv
!   { String CharCount CharSize getinterval ShowChar
!     /CharCount CharCount CharSize add def
    } repeat
  } def
